advanced_alchemy/base.py

```diff
@@ -334,73 +350,148 @@ def create_registry(
 
 
 class UUIDBase(UUIDPrimaryKey, CommonTableAttributes, DeclarativeBase):
-    """Base for all SQLAlchemy declarative models with UUID primary keys."""
+    """Base for all SQLAlchemy declarative models with UUID primary keys.
+
+    See Also:
```

Comment:
```suggestion
    .. seealso::
```
I might would replace this with the sphinx directive so the admonition renders pretty